lib/money/money.rb

```diff
@@ -13,6 +13,16 @@ class Money
   class << self
     attr_accessor :parser, :default_currency
 
+    def opt_in_v1?
+      @opt_in_v1
+    end
+    def opt_in_v1=(value)
+      @opt_in_v1 = value
+      if value
+        Money.active_support_deprecator.behavior = :raise
```

Comment:
I think we should set this to `:raise` and point people towards either https://blog.testdouble.com/posts/2020-12-15-adding-disallowed-deprecations-to-rails-6-1/ or https://github.com/Shopify/deprecation_toolkit to manage this - no need to reinvent the wheel.

Reply:
Are you saying we should raise by default? 🤔 Not opposed to the idea. I was thinking we'd do this in three steps: 1) optional opt-in 2) opt-in by default, with option to opt-out 3) you're on your own... 😅

Reply:
We already have given the people the ability to optionally opt-in to raising instead of logging over the past couple of years 😉 Some apps have done this but the majority hasn't. Changing the default to `:raise` is in effect the the same as 2) opt-in by default.

Reply:
I hear you, I think that opt-in option was not well documented so highly possible not many took advantage of it. That being said I agree with you, the time has come for sane defaults. I'll modify this PR to make it opt-out